tailwind.config.cjs

```diff
@@ -37,6 +37,7 @@ const config = {
 		},
 	},
 	plugins: [require('@tailwindcss/typography'), require('@tailwindcss/forms')],
+	important: true,
```

Comment:
il me semble que si tu prefixe la classe par `!` il rajoute un important

Reply:
Du coup, c'est mieux de l'avoir globalement aulieu de rajotuer ! a chaque fois non ? puisque quand on met une classe tailwind on veut de toute façon qu'elle passe en prio

Reply:
bah si une classe tailwind/bootstrap n'est pas "importante" on ne l'utilise pas tout simplement lol non ? genre si tu mets une "p-0" on part du principe qu'on veut appliquer son style dans tous les cas (responsive y compris).
D'ailleurs, le fait de ne pas le forcer risque même de créer des reg de style plus tard. Par ex, si un "p-0" passe bien sans le "!" sur un element donné, et que demain ce meme element hérite d'une maniere ou d'une autre un nouveau padding, on perdra la prop de la classe tailwind sans forcement se rendre compte. (cf. screen)
<img width="923" alt="image" src="https://user-images.githubusercontent.com/31065028/167611879-91f88b27-2ae2-4f11-aff7-327fd276736d.png">

!important est une très mauvaise pratique certes, on ne devrait pas en avoir dans notre propre code, mais dans le cas de lib utilitaires c'est vital (Bootstrap, tailwind etc): "We enable this setting because utilities should always have the final say."